### PR TITLE
README, aurora-pg.yaml and Banner updates

### PR DESCRIPTION
**Pull request by kmsarabu**
*Issue #, if available:*

*Description of changes:*

1. README update
2. aurora-pg.yaml update: `minCapacity` to 0.5 & `maxCapacity` to 4
3. Banner update

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by cdaveaws — COMMENTED** [review]
I think we should not talk about manual install, just show how to configure flux and install db and app

File: README.md [hunk 1]
```diff
@@ -1,92 +1,229 @@
-# ACK-GitOps-Workshop
+# DAT312: Implement GitOps and manage your Amazon RDS resources using ACK from K8s
 
-DAT312: This is for Invent 2022: Databases Workshop [Implement GitOps and manage your Amazon RDS resources using ACK]
+In this workshop, learn to deploy a continuous integration and delivery (CI/CD) workflow using GitOps and the AWS Controllers for Kubernetes (ACK) service controller for Amazon RDS on Amazon EKS to create and manage Amazon Aurora Serverless v2 databases effectively. GitOps relies on Git as the single source of truth for declaratively managing containerized infrastructure and application components. With Git at the center of CI/CD pipelines, developers can automate and simplify application deployments and operations to Kubernetes. You must bring your laptop to participate.
 
 ## Getting started
 
-To make it easy for you to get started with GitLab, here's a list of recommended next steps.
+For this example we assume a scenario with two clusters: dev and production. The end goal is to leverage Flux and Kustomize to manage both clusters while minimizing duplicated declarations.
```
[the remark this answers is not in the document]
Comment (kmsarabu) [reply README.md:7]: For blue/green deployment strategy

File: apps/src/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# A Demo Retail Application micro services using PostgreSQL & Redis as backend
+
+This [project](https://github.com/aws-samples/ack-rds-gitops-workshop) uses the psycopg2 library to connect to PostgreSQL databases. Psycopg2 is released under the LGPL-3.0 license.
```
Comment (jkatz) [inline apps/src/README.md:3]:
```suggestion
This [project](https://github.com/aws-samples/ack-rds-gitops-workshop) uses the [psycopg2](https://github.com/psycopg/psycopg2) library to connect to PostgreSQL databases. [psycopg2](https://github.com/psycopg/psycopg2) is released under the LGPL-3.0 license.
```

**Comment by kmsarabu** [comment]
> 

I am afraid that the Flux steps alone will not work without the pre-reqs.

**Review by jkatz — APPROVED** [review]

**Review by ajrajkumar — APPROVED** [review]

**Pull request merged**